internal/characters/cyno/burst.go

```diff
@@ -38,10 +42,14 @@ func (c *char) Burst(p map[string]int) action.ActionInfo {
 			return m, true
 		},
 	})
+	c.burstSrc = c.Core.F
+	// if cyno extends his burst, we need to set skill CD properly
+	for i := 1; i < 3; i++ {
+		c.QueueCharTask(func() { c.onBurstExpiry(c.burstSrc) }, 713+i*240)
```

Comment:
I'd prefer if this was more explicit, like 2 lines with 713+240 and 713+480 as delay.